specificationRepositoryConfiguration.json

```diff
@@ -3,42 +3,42 @@
   "pipelineName": "automation - sdk",
   "sdkRepositoryMappings": {
     "azure-sdk-for-go": {
-      "integrationRepository": "AzureSDKAutomation/azure-sdk-for-go",
+      "integrationRepository": "azure-sdk/azure-sdk-for-go",
       "mainRepository": "Azure/azure-sdk-for-go"
     },
     "azure-sdk-for-go-track2": {
-      "integrationRepository": "AzureSDKAutomation/azure-sdk-for-go",
+      "integrationRepository": "azure-sdk/azure-sdk-for-go",
       "mainRepository": "Azure/azure-sdk-for-go",
       "configFilePath": "eng/swagger_to_sdk_config.json"
     },
     "azure-sdk-for-java": {
-      "integrationRepository": "AzureSDKAutomation/azure-sdk-for-java",
+      "integrationRepository": "azure-sdk/azure-sdk-for-java",
       "mainRepository": "Azure/azure-sdk-for-java",
       "configFilePath": "eng/mgmt/automation/swagger_to_sdk_config.json"
     },
     "azure-sdk-for-js": {
-      "integrationRepository": "AzureSDKAutomation/azure-sdk-for-js",
+      "integrationRepository": "azure-sdk/azure-sdk-for-js",
       "mainRepository": "Azure/azure-sdk-for-js"
     },
     "azure-sdk-for-net": {
-      "integrationRepository": "AzureSDKAutomation/azure-sdk-for-net",
+      "integrationRepository": "azure-sdk/azure-sdk-for-net",
       "mainRepository": "Azure/azure-sdk-for-net"
     },
     "azure-sdk-for-net-track2": {
-      "integrationRepository": "AzureSDKAutomation/azure-sdk-for-net",
+      "integrationRepository": "azure-sdk/azure-sdk-for-net",
       "mainRepository": "Azure/azure-sdk-for-net",
       "configFilePath": "eng/swagger_to_sdk_config.json"
     },
     "azure-sdk-for-python-track2": {
-      "integrationRepository": "AzureSDKAutomation/azure-sdk-for-python",
+      "integrationRepository": "azure-sdk/azure-sdk-for-python",
       "mainRepository": "Azure/azure-sdk-for-python"
     },
     "azure-resource-manager-schemas": {
```

Comment:
As ARM schema own by Lei, @leni-msft to confirm

Reply:
@dw511214992 What's the purpose of changing integration repo owner?

Reply:
@weshaggard 's team is working on to moving all automation work to bot `azure-sdk`.